languages/go/README.md

```diff
@@ -80,34 +80,34 @@ project, err := client.Projects().Delete([]string{"project_id_1", "project_id_2"
 #### Create a Secret
 
 ```go
-secret, err := client.Secrets().Create("key", "value", "note", "organization_id", []string{"project_id"})
+secret, err := bitwardenClient.Secrets().Create("key", "value", "note", "organization_id", []string{"project_id"})
 ```
 
 #### List Secrets
```

Comment:
⛏️ Should we add Get Secrets (by id) and Get (multiple) Secrets By Ids ?

Reply:
Added in 47f059e.